aptos-move/block-executor/src/counters.rs

```diff
@@ -22,12 +24,52 @@ pub static SPECULATIVE_ABORT_COUNT: Lazy<IntCounter> = Lazy::new(|| {
     .unwrap()
 });
 
-/// Count of times a transaction got suspended due to an estimated r/w dependency.
-pub static DEPENDENCY_SUSPEND_COUNT: Lazy<IntCounter> = Lazy::new(|| {
-    register_int_counter!(
-        "aptos_execution_dependency_suspend_count",
-        "Count write estimates encountered when reading in parallel execution \
-        (typically leading to a suspension / waiting)"
+pub static VM_INIT_SECONDS: Lazy<Histogram> = Lazy::new(|| {
+    register_histogram!(
+        // metric name
+        "aptos_execution_vm_init_seconds",
+        // metric description
+        "The time spent in seconds in initializing the VM in the block executor",
+        exponential_buckets(/*start=*/ 1e-3, /*factor=*/ 2.0, /*count=*/ 20).unwrap(),
+    )
+    .unwrap()
+});
+
+pub static TASK_VALIDATE_SECONDS: Lazy<Histogram> = Lazy::new(|| {
```

Comment:
should be fine, but validation is actually really fast, and can we check p2p benchmark with and without this one?

p2p because any overhead might be hidden by other parts of the system and will def not show up on executor / forge.

to run aptos-move/aptos-transactional-benchmarks and cargo bench peer_to_peer

Reply:
Ran the P2P benchmark with and without this counter and didn't see any difference. 

```
With counters - 

peer_to_peer            time:   [102.38 ms 102.61 ms 102.79 ms]
                        change: [-1.6977% -0.8846% -0.3286%] (p = 0.02 < 0.05)
                        Change within noise threshold.

Benchmarking peer_to_peer_parallel: Warming up for 3.0000 s
Warning: Unable to complete 10 samples in 5.0s. You may wish to increase target time to 5.1s or enable flat sampling.
peer_to_peer_parallel   time:   [51.652 ms 51.990 ms 52.244 ms]
                        change: [-3.4084% -1.8709% +0.1246%] (p = 0.05 > 0.05)
                        No change in performance detected.


Without counters - 

peer_to_peer            time:   [102.94 ms 103.98 ms 105.44 ms]
                        change: [+0.3239% +1.4549% +2.6823%] (p = 0.02 < 0.05)
                        Change within noise threshold.
Found 1 outliers among 10 measurements (10.00%)
  1 (10.00%) high mild

Benchmarking peer_to_peer_parallel: Warming up for 3.0000 s
Warning: Unable to complete 10 samples in 5.0s. You may wish to increase target time to 5.2s or enable flat sampling.
peer_to_peer_parallel   time:   [52.847 ms 53.300 ms 53.629 ms]
                        change: [-15.939% -12.201% -7.7295%] (p = 0.00 < 0.05)
                        Performance has improved.
Found 1 outliers among 10 measurements (10.00%)
  1 (10.00%) high severe
```